Makefile

```diff
@@ -183,15 +183,9 @@ test-exec-engine:
 # test vectors.
 test: test-release
 
-# Updates the CLI help text pages in the Lighthouse book, building with Docker.
-cli:
```

Comment:
I think this mainly exists to help Windows users who can't execute the bash script locally. Maybe we could just change CI to call `make cli-local`?

Reply:
Yes the docker command is to support Windows users. I just tested that it is still working in Windows. 

Agree about changing the CI from `make cli` to `make cli-local` here:

https://github.com/sigp/lighthouse/blob/2e0eb6d1b8705bbda2ba56eb195d9cc7c6575e95/.github/workflows/test-suite.yml#L398

Reply:
Ah I see. I've updated CI to use `cli-local`